examples/jsm/nodes/accessors/NormalNode.js

```diff
@@ -37,7 +37,17 @@ class NormalNode extends Node {
 
 		if ( scope === NormalNode.GEOMETRY ) {
 
-			outputNode = attribute( 'normal', 'vec3' );
+			const geometryAttribute = builder.hasGeometryAttribute( 'normal' );
+
+			if ( geometryAttribute === false ) {
+
+				outputNode = attribute( 'position', 'vec3' );
```

Comment:
I don't understand how this can work correctly, wouldn't it be better to use `material.flatShading`?

Reply:
If the normal attribute is unavailable, the `NormalNode` will return `0,0,0`, causing all shading to break. Now, I default the normal value to what seems to be a standard in such cases: `outputNode = vec3( 0, 1, 0 );` as it makes more sense.

Reply:
Updated the demo, looks a lot better that way indeed.